app/components/dashboard/ProgressGraph.js

```diff
@@ -114,32 +114,35 @@ const PainLabel = (props) => (
 export default ({ treatments, width, height }) => {
   const chartWidth = width ? width : 800;
   const chartHeight = height ? height : 500;
+
   return (
-    <LineChart className= "progress-graph"
-      width={ chartWidth } height={ chartHeight }
-      data={fakeData(treatments)}
-      margin={{ top: 20, right: 30, left: 30, bottom: 10 }}>
-      <XAxis dataKey="date" height={100} tick={<CustomizedDateTick/>}
-        interval={0} tickCount={7} />
-      <YAxis label={<PainLabel chartHeight={ chartHeight } />} type="number"
-        domain={[0, 5]} interval={0} tickCount={6} tick={<CustomizedPainTick/>} />
-      <CartesianGrid strokeDasharray="3 3" />
-      <Tooltip/>
-      <Legend verticalAlign="top" iconSize={16} height={30} />
-      {
-        treatments.map(treatment => {
-          let lineColor = randomColor({ luminosity: 'dark', hue: 'rgb' });
-          return (
-            <Line key={treatment.id}
-              type="monotone" legendType="square"
-              connectNulls={ true }
-              dataKey={ treatment.exercise.title }
-              stroke={lineColor}
-              dot={<CustomizedDot label={treatment.exercise.title} />}
-              activeDot={{ stroke: lineColor, r: 10, opacity: .4 }} />
-          )
-        })
-      }
-  </LineChart>
+    <div className= "progress-graph">
+      <ResponsiveContainer width='100%' minHeight={ chartHeight }>
+        <LineChart data={fakeData(treatments)}
```

Comment:
My aim for the 'final draft' deployment is to only upload 'real' therapists (2), patients (10), and exercises (20). From there, we can use our own UI to make a few treatment plans corresponding to our demo's 'flow' section. If I'm understanding this function correctly, that's fine as far as workouts are concerned and what data will be displayed in the dashboard (i.e., it will still display some good ol' fake data on a chart). Am I correct in my understanding?